packages/scripts/config/webpack.config.js

```diff
@@ -138,6 +139,7 @@ const config = {
 				},
 				extractComments: false,
 			} ),
+			new OptimizeCssAssetsPlugin(),
```

Comment:
It's a webpack plugin, shouldn't it be in the plugins section like in the example provided on GitHub?:

https://github.com/NMFR/optimize-css-assets-webpack-plugin#example